org.gnome.Hitori.json

```diff
@@ -0,0 +1,47 @@
+{
+  "app-id": "org.gnome.Hitori",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
```

Comment:
I wonder if you could use 3.26 instead.

Reply:
I’ve changed it to 3.26 now. Not necessary, but does mean we don’t force the user to download the 3.24 runtime unnecessarily, given that most other apps use 3.26.